src/models/item.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const itemSchema = new Schema({
+  name: {
+    type: String,
+    maxLength: [30, "item name's max length is 30 characters"],
+    required: [true, 'Item name is required'],
+  },
+  type: {
+    type: String,
+    enum: {
+      values: [
+        'Books',
+        'Stationery',
+        'School Books',
+        'Novels',
+        'Test Books',
+        'Furniture',
+      ],
+      message:
+        "Item type should be equal to 'Books', 'Stationery', 'School Books', 'Novels', 'Test Books' or 'Furniture'",
```

Comment:
A better error message could be: `Item type should be any one of 'Books', 'Stationery', 'School Books', 'Novels', 'Test Books' or 'Furniture'`